src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__days days days--29">
+        <div class="days__day days__day--1 days__day--1--start-day_sun"></div>
```

Comment:
![image](https://user-images.githubusercontent.com/110608451/187493876-cf4de98a-7b5b-4fde-b377-defdab51705f.png)

Reply:
But I added numbers using CSS
![image](https://user-images.githubusercontent.com/109616961/187498094-a9d554bc-f17d-42a5-8dd6-c2b445f4ae46.png)

Reply:
I meant for class names

Reply:
Yeah. remove the numbers from days` classes.
Also, modifiers classes must be applied to calendar, not day.